core/iwasm/aot/aot_runtime.c

```diff
@@ -1814,7 +1816,7 @@ aot_invoke_native(WASMExecEnv *exec_env, uint32 func_idx, uint32 argc,
     }
 
     attachment = import_func->attachment;
-    if (import_func->call_conv_wasm_c_api) {
+    if (import_func->call_conv_wasm_c_api && c_api_func_import) {
```

Comment:
Seems no need to change here if the code above is changed to what I mentioned, since if `import_func->call_conv_wasm_c_api && c_api_func_import == NULL`, func_ptr will be NULL, and exception will be thrown in L1814 and then L1815 will exit, L1819 won't be executed.

Reply:
great, changes done -
ty @wenyongh, it looks much better now!